doc/radosgw/frontends.rst

```diff
@@ -95,6 +95,15 @@ Options
 :Type: Integer
 :Default: ``65000``
 
+``enable_keep_alive``
+
+:Description: Optional value to enable connection to keep alive, either
+              "yes" or "no". It allows clients to reuse TCP connections
+              for subsequent HTTP requests, which improves performance.
```

Comment:
this description confuses http keepalive with tcp keepalive

the frontend already supports http keepalive based on the http `Connection` header. see `handle_connection()` in `rgw_asio_frontend.cc`, which loops on `parser.keep_alive()`

this pr is enabling `SO_KEEPALIVE` on the socket, which controls tcp's keepalive. this differs from civetweb's `enable_keep_alive` option which controls http keepalive. is that really the intent here?

Reply:
You're right. I'll close this PR.

Reply:
Hi, I opened the issue https://tracker.ceph.com/issues/48402 and I can say that tcp keepalive is actually the intent here.  So this PR looked like a fix for my issue, only the documentation part is wrong.  Can this be reopened?

Reply:
hi @r0ss3, i added some thoughts to https://tracker.ceph.com/issues/48402, shall we continue the discussion there?